forc-plugins/forc-client/src/op/run.rs

```diff
@@ -119,8 +119,8 @@ async fn send_tx(
     tx: &Transaction,
     pretty_print: bool,
     simulate: bool,
-) -> Result<Vec<fuel_tx::Receipt>> {
-    let id = format!("{:#x}", tx.id());
+) -> Result<Vec<Receipt>> {
+    let id = format!("{:#x}", tx.id(&ConsensusParameters::default()));
```

Comment:
It looks like id now requires `ConsensusParameters` not sure why but I am providing the default. Not sure if that is how we should handle this.